src/components/button/button.scss

```diff
@@ -3,9 +3,6 @@
 
 @use '../../style/internal/lime-theme';
 
-@use '@material/button/styles';
-@use '@material/button';
```

Comment:
In other components like `limel-input-field` we are importing other functionality from MDC. 👇 

https://github.com/Lundalogik/lime-elements/blob/14eca6dce61dbb34519036d0453fee4996cd9b16/src/components/input-field/input-field.tsx#L1

But here, we are just importing some styles.